spring-boot-project/spring-boot-actuator/src/main/java/org/springframework/boot/actuate/context/properties/ConfigurationPropertiesReportEndpoint.java

```diff
@@ -231,10 +233,7 @@ else if (item instanceof List) {
 		@Override
 		public Object findFilterId(Annotated a) {
 			Object id = super.findFilterId(a);
-			if (id == null) {
-				id = CONFIGURATION_PROPERTIES_FILTER_ID;
-			}
-			return id;
+			return (id != null) ? id : CONFIGURATION_PROPERTIES_FILTER_ID;
```

Comment:
This is unrelated to the task at hand, please refrain from polishing code even if it's in the same class.

Reply:
Yes, I will change this back, this is a temporary modification I made to re trigger CI builder. If you have any other methods that can be triggered repeatedly, please let me know.